cmd/mps-control-daemon/main.go

```diff
@@ -110,6 +112,10 @@ func start(c *cli.Context, cfg *Config) error {
 	var restartTimeout <-chan time.Time
 	var daemons []*mps.Daemon
 restart:
+	readyFile := mps.ReadyFile{}
+	if err := readyFile.Remove(); err != nil {
+		klog.Warningf("Failed to remove .ready file: %v", err)
+	}
```

Comment:
What if MPS is turned off in the config? Will a pevious ready file just be sitting around until MPS get reenabled at some point in the future?

Reply:
Yes, with the current implementation. What options do we have to ensure that this is removed if the MPS daemonset is removed?